content/applications/finance/accounting/customer_invoices/terms_conditions.rst

```diff
@@ -20,6 +23,9 @@ the terms and conditions in the text box below.
 .. image:: terms_conditions/terms-note.png
    :alt: Example of terms and conditions as a note
 
+.. tip::
+   Add the terms and conditions as an attachment when sending the document via email.
```

Comment:
Good idea to mention this as a tip, since it's that's not the default use of this feature.

```suggestion
   You can also add a PDF version of your terms and conditions as an attachment when sending the document via email. Edit the email templates if you want to include them by default.
```